aws/pr_review/iam.tf

```diff
@@ -21,6 +21,32 @@ data "aws_iam_policy_document" "forms_lambda_client" {
   }
 }
 
+data "aws_iam_policy_document" "forms_lambda_parameter_store" {
+  statement {
+    actions = ["ssm:GetParameters"]
+    effect = "Allow"
+    resources = [
+      "arn:aws:ssm:ca-central-1:687401027353:parameter/form-viewer/env"
+    ]
+  }
+}
+
+resource "aws_iam_policy" "forms_lambda_parameter_store" {
+  name   = "formsLambdaParameterStoreRetrieval"
+  path   = "/"
+  policy = data.aws_iam_policy_document.forms_lambda_parameter_store.json
+
+  tags = {
+    (var.billing_tag_key) = var.billing_tag_value
+    Terraform             = true
+  }
+}
```

Comment:
Minor tweak - could you add the `var.env == "staging" ? 1 : 0` logic to both of these so they don't get created in prod.